Ginger/Ginger/ApplicationModelsLib/POMModels/POMWizardLib/LearnWizard/POMGeneralDetailsWizardPage.xaml

```diff
@@ -30,10 +32,20 @@
                     <Ginger:ucTagsViewer x:Name="xTagsViewer" HorizontalAlignment="Left" Margin="-5,0,0,0"/>
                 </StackPanel>
 
-                <StackPanel Margin="0,5,0,0" >
-                    <Label Content="Page URL:" Style="{StaticResource $LabelStyle}"/>
-                    <TextBox x:Name="xURLTextBox" AutomationProperties.AutomationId="PageURL POMID" Style="{StaticResource $TextBoxStyle}" FontWeight="Bold"  />
+                <StackPanel Margin="0,0,0,0" HorizontalAlignment="Left" >
+                    <Label Margin="5,0,0,0" Style="{StaticResource $LabelStyle}" Content="Page Load Flow:" FontWeight="Bold" HorizontalAlignment="Left" VerticalAlignment="Center"/>
+                    <RadioButton x:Name="xRDBPageURL" Height="20" Content="Page URL"  Grid.Column="0" Grid.Row="1" Tag="radPageURL" Checked="XRDBPageURL_Checked" HorizontalAlignment="Left" Margin="10,5,0,0" />
+                    <RadioButton x:Name="xRDBBF" Height="20"  Grid.Column="0" Grid.Row="1" Tag="radPageURL" HorizontalAlignment="Left" Margin="10,5,0,0" Checked="XRDBBF_Checked" >
+                        <RadioButton.Content>
+                            <GingerCore:ucTextDicResource Text="[BusinessFlow]"/>
+                        </RadioButton.Content>
+                    </RadioButton>
                 </StackPanel>
+                
+                <StackPanel Margin="150,-43,0,0" x:Name="xPageUrlStackPanel" >
```

Comment:
having Margin like "150,-43,0,0" means something is wrong with layout design

Reply:
tried to handle with stackpanel but since we have to hide and show some controls